<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Human Pangenome</title>
<meta name="keywords" content="">
<meta name="description" content="Publications">
<meta name="author" content="">
<link rel="canonical" href="/publications/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.4e77599b937c6398f97d67eb77f162a5f923e80a57e8724b3e58c1174c22bc14.css" integrity="sha256-TndZm5N8Y5j5fWfrd/Fipfkj6ApX6HJLPljBF0wivBQ=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.c5dbfedede2988504e5bcc0fac0383a7a0f77931bdaf40c90feea1f14fc98150.js" integrity="sha256-xdv&#43;3t4piFBOW8wPrAODp6D3eTG9r0DJD&#43;6h8U/JgVA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
<link rel="apple-touch-icon" href="apple-touch-icon.png">
<link rel="mask-icon" href="safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="" />
<meta property="og:description" content="Publications" />
<meta property="og:type" content="article" />
<meta property="og:url" content="/publications/" /><meta property="article:section" content="menu" />



<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content=""/>
<meta name="twitter:description" content="Publications"/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Menus",
      "item": "/menu/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "",
      "item": "/publications/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "",
  "name": "",
  "description": "Publications",
  "keywords": [
    
  ],
  "articleBody": " There are several HPRC publications in development at this time. Please check back for updates and follow us on Twitter @HumanPangenome. Below is a selection of publications from the HPRC membership.\nTitle Authors DOI Date Pangenome-based genome inference allows efficient and accurate genotyping across a wide spectrum of variant classes. Ebler J, Ebert P, Clarke WE, Rausch T, Audano PA, Houwaart T, Mao Y, Korbel JO, Eichler EE, Zody MC, Dilthey AT, Marschall T. 10.1038/s41588-022- 01043-w 2022-04-11 Concerted modification of nucleotides at functional centers of the ribosome revealed by single-molecule RNA modification profiling. Bailey AD, Talkish J, Ding H, Igel H, Duran A, Mantripragada S, Paten B, Ares M.Elife 10.7554/eLife.76562 2022-04-06 Segmental duplications and their variation in a complete human genome. Vollger MR, Guitart X, Dishuck PC, et al. 10.1126/science.abj69 65 2022-04-01 StainedGlass: Interactive visualization of massive tandem repeat structures with identity heatmaps. Vollger MR, Kerpedjiev P, Phillippy AM, Eichler EE. 10.1093/bioinformatics /btac018 2022-01-10 The UCSC Genome Browser database: 2022 update. Lee BT, Barber GP, Benet-Pag√®s A, et al. 10.1093/nar/gkab959 2022-01-07 Profiling variable-number tandem repeat variation across populations using repeat-pangenome graphs Tsung-Yu Lu, Human Genome Structural Variation Consortium, Mark Chaisson. 10.1038/s41467-021- 24378-0 2021-07-12 Targeted long-read sequencing identifies missing disease-causing variation Miller, D., Sulovari, A., Wang, T., et al. 10.1016/j.ajhg.2021.06 .006 2021-07-02 lra: A long read aligner for sequences and contigs Jingwen Ren, Mark Chaisson. 10.1371/journal.pcbi.10 09078 2021-06-21 The qBED track: a novel genome browser visualization for point processes Moudgil, A., Li, D., Hsu, S., et al. 10.1093/bioinformatics /btaa771 2021-05-23 Common DNA methylation dynamics in endometriod adenocarcinoma and glioblastoma suggest universal epigenomic alterations in tumorigenesis Karlow, J., Miao, B., Xing, X., et al. 10.1038/s42003-021- 02094-1 2021-05-21 A high-quality bonobo genome refines the analysis of hominid evolution Mao, Y., Catacchio, C., Hillier, L., et al. 10.1038/s41586-021- 03519-x 2021-05-05 The Need for a Human Pangenome Reference Sequence Miga, K., Wang, T 10.1146/annurev-gen om-120120-081921 2021-04-30 Human placental cytotrophoblast epigenome dynamics over gestation and alterations in placental disease Zhang, B., Kim, B., Elliot, G., et al. 10.1016/j.devcel.2021 .04.001 2021-04-20 Profiling SARS-CoV-2 mutation fingerprints that range from the viral pangenome to individual infection quasispecies Lau, B., Pavlichin, D., Hooker, A., et al. 10.1186/s13073-021- 00882-2 2021-04-19 The structure, function and evolution of a complete human chromosome 8 Logsdon, G., Vollger, M., Hsieh, P., et al. 10.1038/s41586-021- 03420-7 2021-04-07 Haplotype-resolved diverse human genomes and integrated analysis of structural variation Ebert, P., Audano, P., Zhu, Q., et al. 10.1126/science.abf7 117 2021-02-25 Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm Cheng, H., Concepcion, G., Feng, X., et al. 10.1038/s41592-020- 01056-5 2021-02-01 Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads Porubsky, D., et al. 10.1038/s41587-020- 0719-5 2020-12-07 Chromosome-scale, haplotype-resolved assembly of human genomes Garg, S., Fungtammasan, A., Carroll, A., et al. 10.1038/s41587-020- 0711-0 2020-12-07 A map of cis-regulatory elements and 3D genome structures in zebrafish Yang, H., Luan, Y., Liu, T., et al. 10.1038/s41586-020- 2962-9 2020-11-25 The design and construction of reference pangenome graphs with minigraph Li, H., Feng, X., Chu, C. 10.1186/s13059-020- 02168-z 2020-10-16 Cell-type-specific 3D epigenomes in the developing human cortex Song, M., Pebworth, M., Yang, X., et al. 10.1038/s41586-020- 2825-4 2020-10-14 Tissue-specific usage of transposable element-derived promoters in mouse development Miao, B., Fa, S., Lyu, C., et al. 10.1186/s13059-020- 02164-3 2020-09-28 Telomere-to-telomere assembly of a complete human X chromosome Miga, K., Koren, S., Rhie, A. et al. 10.1038/s41586-020- 2547-7 2020-07-14 Epigenomics and genotype-phenotype association analyses reveal conserved genetic architecture of complex traits in cattle and human Liu, S., Yu, Y., Zhang, S., et al. 10.1186/s12915-020- 00792-6 2020-07-03 The design and construction of reference pangenome graphs Li, H., Feng, X., Chu, C. arXiv:2003.06079 2020-03-13 Genotyping structural variants in pangenome graphs using the vg toolkit Hickey, G., Heller, D., Monlong, J., et al. 10.1186/s13059-020- 1941-7 2020-02-12 A fully phased accurate assembly of an individual human genome Porubsky, D., Ebert, P., Audano, P., et al. 10.1101/855049 2019-11-26 Efficient de novo assembly of eleven human genomes using PromethION sequencing and a novel nanopore toolkit Shafin, K., Pesout, T., Lorig-Roach, R., et al. 10.1101/715722 2019-07-26 Progressive alignment with Cactus: a multiple-genome aligner for the thousand-genome era Armstrong, J., Hickey, G., Diekhans, M., et al. 10.1101/730531 2019-10-09 ",
  "wordCount" : "692",
  "inLanguage": "en",
  "datePublished": "0001-01-01T00:00:00Z",
  "dateModified": "0001-01-01T00:00:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "/publications/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Human Pangenomes",
    "logo": {
      "@type": "ImageObject",
      "url": "favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://eshi1.github.io/" accesskey="h" title="Human Pangenome (Alt + H)">
                <img src="/img/Logo.png" alt="logo" aria-label="logo"
                    height="40">Human Pangenome</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
            <li>
                <a href="/aboutus/" title="About Us">
                    <span>About Us</span>
                </a>
            </li>
            <li>
                <a href="/data/" title="Data &amp; Resources">
                    <span>Data &amp; Resources</span>
                </a>
            </li>
            <li>
                <a href="/events/" title="Events &amp; Presentations">
                    <span>Events &amp; Presentations</span>
                </a>
            </li>
            <li>
                <a href="/publications/" title="Publications">
                    <span class="active">Publications</span>
                </a>
            </li>
            <li>
                <a href="/contact/" title="Contact Us">
                    <span>Contact Us</span>
                </a>
            </li>
        </ul>
    </nav>
</header>
<main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      
    </h1>
    <div class="post-meta">

</div>
  </header> 
  <div class="post-content"><style> 
.test{
    animation: transitionIn 1s;
}
@keyframes transitionIn{
    from{
        opacity:0;
        transform: rotateX(-10deg);}
    to{
        opacity:1;
        transform: rotateX(0);}
}
</style>

<div class="test">
<img src="/img/p3.png">
</div>
<p>There are several HPRC publications in development at this time. Please check back for updates and follow us on Twitter <a href="https://twitter.com/HumanPangenome" target="_blank">@HumanPangenome</a>.  Below is a selection of publications from the HPRC membership.</p>
<table>
<thead>
<tr>
<th>Title</th>
<th>Authors</th>
<th>DOI</th>
<th>Date</th>
</tr>
</thead>
<tbody>
<tr>
<td>Pangenome-based genome inference allows efficient and accurate genotyping across a wide spectrum of variant classes.</td>
<td>Ebler J, Ebert P, Clarke WE, Rausch T, Audano PA, Houwaart T, Mao Y, Korbel JO, Eichler EE, Zody MC, Dilthey AT, Marschall T.</td>
<td><a href="https://www.nature.com/articles/s41588-022-01043-w" target="_blank">10.1038/s41588-022- 01043-w</a></td>
<td>2022-04-11</td>
</tr>
<tr>
<td>Concerted modification of nucleotides at functional centers of the ribosome revealed by single-molecule RNA modification profiling.</td>
<td>Bailey AD, Talkish J, Ding H, Igel H, Duran A, Mantripragada S, Paten B, Ares M.Elife</td>
<td><a href="https://elifesciences.org/articles/76562" target="_blank">10.7554/eLife.76562</a></td>
<td>2022-04-06</td>
</tr>
<tr>
<td>Segmental duplications and their variation in a complete human genome.</td>
<td>Vollger MR, Guitart X, Dishuck PC, et al.</td>
<td><a href="https://www.science.org/doi/10.1126/science.abj6965" target="_blank">10.1126/science.abj69 65</a></td>
<td>2022-04-01</td>
</tr>
<tr>
<td>StainedGlass: Interactive visualization of massive tandem repeat structures with identity heatmaps.</td>
<td>Vollger MR, Kerpedjiev P, Phillippy AM, Eichler EE.</td>
<td><a href="https://pubmed.ncbi.nlm.nih.gov/35020798/" target="_blank">10.1093/bioinformatics /btac018</a></td>
<td>2022-01-10</td>
</tr>
<tr>
<td>The UCSC Genome Browser database: 2022 update.</td>
<td>Lee BT, Barber GP, Benet-Pag√®s A, et al.</td>
<td><a href="https://pubmed.ncbi.nlm.nih.gov/34718705/" target="_blank">10.1093/nar/gkab959</a></td>
<td>2022-01-07</td>
</tr>
<tr>
<td>Profiling variable-number tandem repeat variation across populations using repeat-pangenome graphs</td>
<td>Tsung-Yu Lu, Human Genome Structural Variation Consortium, Mark Chaisson.</td>
<td><a href="https://doi.org/10.1038/s41467-021-24378-0" target="_blank">10.1038/s41467-021- 24378-0</a></td>
<td>2021-07-12</td>
</tr>
<tr>
<td>Targeted long-read sequencing identifies missing disease-causing variation</td>
<td>Miller, D., Sulovari, A., Wang, T., et al.</td>
<td><a href="https://doi.org/10.1016/j.ajhg.2021.06.006" target="_blank">10.1016/j.ajhg.2021.06 .006</a></td>
<td>2021-07-02</td>
</tr>
<tr>
<td>lra: A long read aligner for sequences and contigs</td>
<td>Jingwen Ren, Mark Chaisson.</td>
<td><a href="https://doi.org/10.1371/journal.pcbi.1009078" target="_blank">10.1371/journal.pcbi.10 09078</a></td>
<td>2021-06-21</td>
</tr>
<tr>
<td>The qBED track: a novel genome browser visualization for point processes</td>
<td>Moudgil, A., Li, D., Hsu, S., et al.</td>
<td><a href="https://doi.org/10.1093/bioinformatics/btaa771" target="_blank">10.1093/bioinformatics /btaa771</a></td>
<td>2021-05-23</td>
</tr>
<tr>
<td>Common DNA methylation dynamics in endometriod adenocarcinoma and glioblastoma suggest universal epigenomic alterations in tumorigenesis</td>
<td>Karlow, J., Miao, B., Xing, X., et al.</td>
<td><a href="https://doi.org/10.1038/s42003-021-02094-1" target="_blank">10.1038/s42003-021- 02094-1</a></td>
<td>2021-05-21</td>
</tr>
<tr>
<td>A high-quality bonobo genome refines the analysis of hominid evolution</td>
<td>Mao, Y., Catacchio, C., Hillier, L., et al.</td>
<td><a href="https://doi.org/10.1038/s41586-021-03519-x" target="_blank">10.1038/s41586-021- 03519-x</a></td>
<td>2021-05-05</td>
</tr>
<tr>
<td>The Need for a Human Pangenome Reference Sequence</td>
<td>Miga, K., Wang, T</td>
<td><a href="https://doi.org/10.1146/annurev-genom-120120-081921" target="_blank">10.1146/annurev-gen om-120120-081921</a></td>
<td>2021-04-30</td>
</tr>
<tr>
<td>Human placental cytotrophoblast epigenome dynamics over gestation and alterations in placental disease</td>
<td>Zhang, B., Kim, B., Elliot, G., et al.</td>
<td><a href="https://doi.org/10.1016/j.devcel.2021.04.001" target="_blank">10.1016/j.devcel.2021 .04.001</a></td>
<td>2021-04-20</td>
</tr>
<tr>
<td>Profiling SARS-CoV-2 mutation fingerprints that range from the viral pangenome to individual infection quasispecies</td>
<td>Lau, B., Pavlichin, D., Hooker, A., et al.</td>
<td><a href="https://doi.org/10.1186/s13073-021-00882-2" target="_blank">10.1186/s13073-021- 00882-2</a></td>
<td>2021-04-19</td>
</tr>
<tr>
<td>The structure, function and evolution of a complete human chromosome 8</td>
<td>Logsdon, G., Vollger, M., Hsieh, P., et al.</td>
<td><a href="https://doi.org/10.1038/s41586-021-03420-7" target="_blank">10.1038/s41586-021- 03420-7</a></td>
<td>2021-04-07</td>
</tr>
<tr>
<td>Haplotype-resolved diverse human genomes and integrated analysis of structural variation</td>
<td>Ebert, P., Audano, P., Zhu, Q., et al.</td>
<td><a href="https://doi.org/10.1126/science.abf7117" target="_blank">10.1126/science.abf7 117</a></td>
<td>2021-02-25</td>
</tr>
<tr>
<td>Haplotype-resolved de novo assembly using phased assembly graphs with hifiasm</td>
<td>Cheng, H., Concepcion, G., Feng, X., et al.</td>
<td><a href="https://doi.org/10.1038/s41592-020-01056-5" target="_blank">10.1038/s41592-020- 01056-5</a></td>
<td>2021-02-01</td>
</tr>
<tr>
<td>Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads</td>
<td>Porubsky, D., et al.</td>
<td><a href="https://doi.org/10.1038/s41587-020-0719-5" target="_blank">10.1038/s41587-020- 0719-5</a></td>
<td>2020-12-07</td>
</tr>
<tr>
<td>Chromosome-scale, haplotype-resolved assembly of human genomes</td>
<td>Garg, S., Fungtammasan, A., Carroll, A., et al.</td>
<td><a href="https://doi.org/10.1038/s41587-020-0711-0" target="_blank">10.1038/s41587-020- 0711-0</a></td>
<td>2020-12-07</td>
</tr>
<tr>
<td>A map of cis-regulatory elements and 3D genome structures in zebrafish</td>
<td>Yang, H., Luan, Y., Liu, T., et al.</td>
<td><a href="https://doi.org/10.1038/s41586-020-2962-9" target="_blank">10.1038/s41586-020- 2962-9</a></td>
<td>2020-11-25</td>
</tr>
<tr>
<td>The design and construction of reference pangenome graphs with minigraph</td>
<td>Li, H., Feng, X., Chu, C.</td>
<td><a href="https://doi.org/10.1186/s13059-020-02168-z" target="_blank">10.1186/s13059-020- 02168-z</a></td>
<td>2020-10-16</td>
</tr>
<tr>
<td>Cell-type-specific 3D epigenomes in the developing human cortex</td>
<td>Song, M., Pebworth, M., Yang, X., et al.</td>
<td><a href="https://doi.org/10.1038/s41586-020-2825-4" target="_blank">10.1038/s41586-020- 2825-4</a></td>
<td>2020-10-14</td>
</tr>
<tr>
<td>Tissue-specific usage of transposable element-derived promoters in mouse development</td>
<td>Miao, B., Fa, S., Lyu, C., et al.</td>
<td><a href="https://doi.org/10.1186/s13059-020-02164-3" target="_blank">10.1186/s13059-020- 02164-3</a></td>
<td>2020-09-28</td>
</tr>
<tr>
<td>Telomere-to-telomere assembly of a complete human X chromosome</td>
<td>Miga, K., Koren, S., Rhie, A. et al.</td>
<td><a href="https://doi.org/10.1038/s41586-020-2547-7" target="_blank">10.1038/s41586-020- 2547-7</a></td>
<td>2020-07-14</td>
</tr>
<tr>
<td>Epigenomics and genotype-phenotype association analyses reveal conserved genetic architecture of complex traits in cattle and human</td>
<td>Liu, S., Yu, Y., Zhang, S., et al.</td>
<td><a href="https://doi.org/10.1186/s12915-020-00792-6" target="_blank">10.1186/s12915-020- 00792-6</a></td>
<td>2020-07-03</td>
</tr>
<tr>
<td>The design and construction of reference pangenome graphs</td>
<td>Li, H., Feng, X., Chu, C.</td>
<td><a href="https://arxiv.org/abs/2003.06079" target="_blank">arXiv:2003.06079</a></td>
<td>2020-03-13</td>
</tr>
<tr>
<td>Genotyping structural variants in pangenome graphs using the vg toolkit</td>
<td>Hickey, G., Heller, D., Monlong, J., et al.</td>
<td><a href="https://doi.org/10.1186/s13059-020-1941-7" target="_blank">10.1186/s13059-020- 1941-7</a></td>
<td>2020-02-12</td>
</tr>
<tr>
<td>A fully phased accurate assembly of an individual human genome</td>
<td>Porubsky, D., Ebert, P., Audano, P., et al.</td>
<td><a href="https://doi.org/10.1101/855049" target="_blank">10.1101/855049</a></td>
<td>2019-11-26</td>
</tr>
<tr>
<td>Efficient de novo assembly of eleven human genomes using PromethION sequencing and a novel nanopore toolkit</td>
<td>Shafin, K., Pesout, T., Lorig-Roach, R., et al.</td>
<td><a href="https://doi.org/10.1101/715722" target="_blank">10.1101/715722</a></td>
<td>2019-07-26</td>
</tr>
<tr>
<td>Progressive alignment with Cactus: a multiple-genome aligner for the thousand-genome era</td>
<td>Armstrong, J., Hickey, G., Diekhans, M., et al.</td>
<td><a href="https://doi.org/10.1101/730531" target="_blank">10.1101/730531</a></td>
<td>2019-10-09</td>
</tr>
</tbody>
</table>


  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>

<div class="share-buttons">
    <a target="_blank"
        href="https://twitter.com/HumanPangenome">
        <svg version="1.1" viewBox="0 0 512 512" xml:space="preserve">
            <path
                d="M449.446,0c34.525,0 62.554,28.03 62.554,62.554l0,386.892c0,34.524 -28.03,62.554 -62.554,62.554l-386.892,0c-34.524,0 -62.554,-28.03 -62.554,-62.554l0,-386.892c0,-34.524 28.029,-62.554 62.554,-62.554l386.892,0Zm-253.927,424.544c135.939,0 210.268,-112.643 210.268,-210.268c0,-3.218 0,-6.437 -0.153,-9.502c14.406,-10.421 26.973,-23.448 36.935,-38.314c-13.18,5.824 -27.433,9.809 -42.452,11.648c15.326,-9.196 26.973,-23.602 32.49,-40.92c-14.252,8.429 -30.038,14.56 -46.896,17.931c-13.487,-14.406 -32.644,-23.295 -53.946,-23.295c-40.767,0 -73.87,33.104 -73.87,73.87c0,5.824 0.613,11.494 1.992,16.858c-61.456,-3.065 -115.862,-32.49 -152.337,-77.241c-6.284,10.881 -9.962,23.601 -9.962,37.088c0,25.594 13.027,48.276 32.95,61.456c-12.107,-0.307 -23.448,-3.678 -33.41,-9.196l0,0.92c0,35.862 25.441,65.594 59.311,72.49c-6.13,1.686 -12.72,2.606 -19.464,2.606c-4.751,0 -9.348,-0.46 -13.946,-1.38c9.349,29.426 36.628,50.728 68.965,51.341c-25.287,19.771 -57.164,31.571 -91.8,31.571c-5.977,0 -11.801,-0.306 -17.625,-1.073c32.337,21.15 71.264,33.41 112.95,33.41Z" />
        </svg>
    </a>
    <a target="_blank" href="https://twitter.com/HumanPangenome"> 
        <pre>Follow Us  </pre>
    </a>
    <a target="_blank"
        href="http://wiki.humanpangenome.org/">
        <svg version="1.1" xml:space="preserve" viewBox="2 2 28 28">
            <path
                d="M26.49,29.86H5.5a3.37,3.37,0,0,1-2.47-1,3.35,3.35,0,0,1-1-2.47V5.48A3.36,3.36,0,0,1,3,3,3.37,3.37,0,0,1,5.5,2h21A3.38,3.38,0,0,1,29,3a3.36,3.36,0,0,1,1,2.46V26.37a3.35,3.35,0,0,1-1,2.47A3.38,3.38,0,0,1,26.49,29.86Zm-5.38-6.71a.79.79,0,0,0,.85-.66L24.73,9.24a.55.55,0,0,0-.18-.46.62.62,0,0,0-.41-.17q-.08,0-16.53,6.11a.59.59,0,0,0-.41.59.57.57,0,0,0,.43.52l4,1.24,1.61,4.83a.62.62,0,0,0,.63.43.56.56,0,0,0,.4-.17L16.54,20l4.09,3A.9.9,0,0,0,21.11,23.15ZM13.8,20.71l-1.21-4q8.72-5.55,8.78-5.55c.15,0,.23,0,.23.16a.18.18,0,0,1,0,.06s-2.51,2.3-7.52,6.8Z" />
        </svg>
    </a>
    <a target="_blank" href="http://wiki.humanpangenome.org/"> 
    <pre> Wiki Login </pre>
    </a>
</div>

  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2022 <a href="">Human Pangenomes</a></span>
    <span>
        Powered by
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">Hugo</a> &
        <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">PaperMod</a>
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
